dagmc/dagnav.py

```diff
@@ -289,10 +300,52 @@ def __init__(self, model: DAGModel, handle: np.uint64):
         super().__init__(model, handle)
         self._check_category_and_dimension()
 
-    def get_volumes(self):
+    @property
+    def surf_sense(self) -> list[np.uint64]:
```

Comment:
Is this currently the only public method that returns MOAB `EntityHandles`? I think it might be better to stick with the `Volume` objects or make this an internal method to stay consistent.

Reply:
Good idea -- I'll update this to use `Volume` instead of handles directly, which also has the benefit of consolidating the checks for 0 handles in one place.